.gitmodules

```diff
@@ -34,3 +34,6 @@
 [submodule "modules/DroneCAN/libcanard"]
 	path = modules/DroneCAN/libcanard
 	url = https://github.com/DroneCAN/libcanard
+[submodule "module/CubeFramework"]
+	path = modules/CubeFramework
+	url = https://github.com/CubePilot/CubeFramework/
```

Comment:
please move to under DroneCAN repo, and I don't think it should use a vendor specific name like 'Cube'

Reply:
done, I moved the wrappers to libcanard now. That seemed the neatest solution.